metapool/sample_sheet.py

```diff
@@ -21,26 +28,31 @@
 SHEET_TYPES = (_STANDARD_METAG_SHEET_TYPE, _ABSQUANT_SHEET_TYPE,
                _STANDARD_METAT_SHEET_TYPE)
 
+_BASE_BIOINFORMATICS_COLS = [
+    'Sample_Project', 'QiitaID', 'BarcodesAreRC', 'ForwardAdapter',
+    'ReverseAdapter', 'HumanFiltering', 'library_construction_protocol',
+    'experiment_design_description']
+_BIOINFORMATICS_COLS_W_REP_SUPPORT = \
+    _BASE_BIOINFORMATICS_COLS + ['contains_replicates']
+_CONTACT_COLS = ['Sample_Project', 'Email']
+_SAMPLE_CONTEXT_COLS = [SAMPLE_NAME_KEY, SAMPLE_TYPE_KEY,
+                        PRIMARY_STUDY_KEY, SECONDARY_STUDIES_KEY]
+
 
 class KLSampleSheet(sample_sheet.SampleSheet):
     _ASSAYS = frozenset({_AMPLICON, _METAGENOMIC, _METATRANSCRIPTOMIC})
     _BIOINFORMATICS_AND_CONTACT = {
-        'Bioinformatics': None,
-        'Contact': None
+        BIOINFORMATICS_KEY: None,
+        CONTACT_KEY: None
     }
 
-    _CONTACT_COLUMNS = frozenset({
-        'Sample_Project', 'Email'
-    })
-
-    _BIOINFORMATICS_COLUMNS = frozenset({
-        'Sample_Project', 'QiitaID', 'BarcodesAreRC', 'ForwardAdapter',
-        'ReverseAdapter', 'HumanFiltering', 'library_construction_protocol',
-        'experiment_design_description'
-    })
-
     _BIOINFORMATICS_BOOLEANS = frozenset({'BarcodesAreRC', 'HumanFiltering'})
 
+    COLS_BY_EXTRA_DF_SECTION_NAME = {
```

Comment:
Agree that the name is confusing.  Could we have any additional sections that *aren't* handled as dataframes?  Because the dict is being used to specify added pieces that specifically get handled *as dataframes*.  Would `KL_ADDTL_DF_SECTIONS` work?

Reply:
Ah I see! KL_ADDTL_DF_SECTIONS works for me, thanks!

Reply:
Done